include/dllshard.hpp

```diff
@@ -130,9 +130,12 @@ class Core {
     return sCore._core->composeShards(shards, callback, userData, data);
   }
 
-  static SHWireState runShards(Shards shards, SHContext *context, const SHVar &input, SHVar &output,
-                               const bool handleReturn = false) {
-    return sCore._core->runShards(shards, context, &input, &output, handleReturn);
+  static SHWireState runShards(Shards shards, SHContext *context, const SHVar &input, SHVar &output) {
+    return sCore._core->runShards(shards, context, &input, &output);
+  }
+
+  static SHWireState runShards2(Shards shards, SHContext *context, const SHVar &input, SHVar &output) {
```

Comment:
Do we need two functions now? They seem to both do the same.

Reply:
If you navigate inside they are, one handles return, one does not (useful in Repeat etc flow blocks).
We probably need a comment pass at some point so that every entry point is covered.

Reply:
check shards.h, It's commented

Reply:
I don't get it. The body of both function is visible here and they both have the exact same line:

```cpp
return sCore._core->runShards(shards, context, &input, &output);
```

So unless the second should call a different function (`sCore._core->runShards2` maybe), they are the same. What am I missing?

Reply:
Oops I see. You were not questioning the functionality 😅
That's a mistake yes ! Thanks 🙏